src/proj_etmerc.c

```diff
@@ -71,8 +71,14 @@ static double gatg(double *p1, int len_p1, double B) {
     double h = 0, h1, h2 = 0, cos_2B;
 
     cos_2B = 2*cos(2*B);
-    for (p = p1 + len_p1, h1 = *--p; p - p1; h2 = h1, h1 = h)
+    p = p1 + len_p1;
+    h1 = *--p;
+    for (; p - p1; )
+    {
```

Comment:
Similar to comment above. Bracket on same line as statement.

Reply:
done